README.md

```diff
@@ -125,11 +125,11 @@ $ cargo test -- --ignored
 
 Language bindings are available for:
 
-- [C](https://c.delta.chat)
-- [Node.js](https://www.npmjs.com/package/deltachat-node)
-- [Python](https://py.delta.chat)
-- [Go](https://github.com/deltachat/go-deltachat/)
-- [Free Pascal](https://github.com/deltachat/deltachat-fp/)
+- **C** \[[📂 source](./deltachat-ffi) | [📚 docs](https://c.delta.chat)\]
+- **Node.js** \[[📂 source](https://github.com/deltachat/deltachat-node) | [📦 npm](https://www.npmjs.com/package/deltachat-node) | [📚 docs](https://js.delta.chat)\]
+- **Python** \[[📂 source](./python) | [📦 pypi](https://pypi.org/project/deltachat) | [📚 docs](https://py.delta.chat)\]
+- **Go** \[[📂 source](https://github.com/deltachat/go-deltachat/)\]
+- **Free Pascal** \[[📂 source](https://github.com/deltachat/deltachat-fp/)\]
 - **Java** and **Swift** (contained in the Android/iOS repos)
```

Comment:
maybe also link to the android and ios repos?

Reply:
hm... they are not usable without changes outside of the android/iOS app. yeah but on the other hand, the go and free pascal bindings are outdated so need work to be updated to be used properly too.

Reply:
The actual Java code at https://github.com/deltachat/deltachat-android/tree/master/src/com/b44t/messenger should be usable without changes outside the android app. There is no Android-specific code there (if there is, it's a mistake and it should be removed).

Building it might be tricky though, you will need the C and C++ files from https://github.com/deltachat/deltachat-android/tree/master/jni and probably need a script similar to https://github.com/deltachat/deltachat-android/blob/master/ndk-make.sh. Idk if someone has already done this.

Reply:
It's not as easy to use the java code in java-land @angelo-fuchs and me already spend a some time on trying to make a maven package for that code, so no it's not so easy usable outside of that context.
maybe we should rather link to https://github.com/angelo-fuchs/dc-jni-playground, but then again it's already a bit outdated and incomplete, the idea was to put it into the core repo after we got it working, but we did not have the time to complete it yet.

So I suggest just merging this pr now and when it's easy to use it from java we can advertise it as such in the readme.

Reply:
yip, let's just merge it